marktplaats/models/listing.py

```diff
@@ -18,11 +20,27 @@ class Listing:
     price: float
     price_type: PriceType
     link: str
-    images: list
+    _images: list[ListingFirstImage]
     category_id: int
     attributes: list
     extended_attributes: list
 
+    @property
+    def images(self) -> list[ListingFirstImage]:
+        warnings.warn("Listing.images is deprecated since marktplaats version 0.3.0. "
+                      "Please use Listing.first_image or Listing.get_images() instead.",
+                      category=DeprecationWarning,
+                      stacklevel=2)
+        return self._images
+
+    @property
+    def first_image(self) -> ListingFirstImage:
+        # marktplaats.nl requires at least 2 images per listing, so we should be safe here
+        return self._images[0]
```

Comment:
This is not true right? The API object can contain no `pictures` element, in which case we get `[]`:
```py
    def parse(cls, data):
        if data is None:
            return []
```
```suggestion
    @property
    def first_image(self) -> ListingFirstImage | None:
        return self._images.get(0)
```

Reply:
I did this because, when creating the ad, it requires at least two images. There might be exceptions though. I agree that we should have a fallback

Reply:
Maybe it varies per category? I've personally seen a lot of listings without images; it's pretty common.